### PR TITLE
Use one Docker image for both pbuilder roots

### PR DESCRIPTION
**Pull request by theucke**
* Install git-buildpackage from jessie-backports to build stretch root
* Put OPX gpg key in /etc/apt/trusted.gpg.d/ to skip apt-key add step
* Touch local Packages file in pbuilder create script

Signed-off-by: Tyler Heucke <tyler.heucke@dell.com>

### CODE REVIEW

**Comment by openswitch-gerrit** [comment]
Thank you for contributing to opx/opx-build!

opx/opx-build uses Gerrit for code review.

Our Gerrit server is located at https://review.openswitch.net

Our Wiki is located at https://github.com/open-switch/opx-docs/wiki

If you have never contributed to OpenStack before make sure you have read the
getting started documentation:
http://docs.openstack.org/infra/manual/developers.html#getting-started

Otherwise please visit
http://docs.openstack.org/infra/manual/developers.html#development-workflow
and follow the instructions there to upload your change to Gerrit.

**Comment by theucke** [comment]
Pull-request updated, HEAD is now 6c3672bf3108d801a7d7d59837baa404bd527a6c

File: Dockerfile-jessie [hunk 1]
```diff
@@ -22,7 +20,7 @@ RUN apt-get -qq update && apt-get -qq upgrade -y && apt-get -qq install -y \
     python-requests \
     vim \
     wget \
- && apt-get -t jessie-backports install -y gosu \
+ && apt-get -t jessie-backports install -y cowbuilder git-buildpackage gosu \
```
Comment (theucke) [inline Dockerfile-jessie]: Testing rebase

**Comment by theucke** [comment]
Pull-request updated, HEAD is now aadbbbff27e518a333b852b63aa08afc5821f895

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ If you would like to download binaries instead, see
 
 ```bash
 # get source code
-repo init -u git://git.openswitch.net/opx/opx-manifest && repo sync
+repo init -u https://github.com/open-switch/opx-manifes && repo sync
```
Comment (theucke) [inline README.md]: Typo. Should be `manifest`

**Comment by theucke** [comment]
Pull-request updated, HEAD is now b52ca54341da023d1397cb42c335edf1ba7ebcc9

**Comment by theucke** [comment]
Pull-request updated, HEAD is now 65831ddc37fd04d2cc796e9219bf0071c2f86718

**Pull request merged**